kotlin/workflow-testing/src/main/java/com/squareup/workflow/testing/WorkflowTester.kt

```diff
@@ -212,11 +222,23 @@ fun <T, InputT, OutputT : Any, RenderingT>
       snapshot: Snapshot? = null,
       context: CoroutineContext = EmptyCoroutineContext,
       block: WorkflowTester<InputT, OutputT, RenderingT>.() -> T
-    ): T = test(block, context) { factory, inputs ->
-      inputs.offer(input)
-      factory.run(this, inputs.asFlow(), snapshot)
-    }
+    ): T
 // @formatter:on
```

Comment:
Did you mean to move the `@formatter:on` above the body?

Reply:
Yea, would like to minimize the amount of code affected by those things.